### ISSUE TITLE
Training code?

### ISSUE THREAD

**Issue by ethanhe42** [ericsun99/Shufflenet-v2-Pytorch-#4]
could you release the training code? Thanks!

**Comment by ialhashim** [comment]
I am also interested in the training code to try and train for x2.

**Comment by dereyly** [comment]
Any suggestions of training params?
I try to train x1.5 with standard 
https://github.com/pytorch/examples/tree/master/imagenet
bs=256, lr=0.1, SGD, weight_decay=1e-4, 
linear decrease of lr -- 90 epoch [30,60]
 and standard augmentation

**Comment by askingbot** [comment]
@ericsun99 I'm also wondering the training hyper-parameters. Could you provide the training script or post the training parameters here? Thank you!

**Comment by pustar** [comment]
could you release the training code or  training hyper-parameters? Thanks!
